carbon-design-system/carbon-for-ibm-dotcom
# fix(cloud-masthead): adjust the Profile icon to 20px

### Related Ticket(s)

GH Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7178
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2200

### Description

- This PR removes un-needed styling for the `:host(#{$dds-prefix}-cloud-masthead-profile)` so that there's no extra padding that makes the icon small.

### Testing Instruction

- Visit the `Cloud Masthead` - `/?path=/story/components-cloud-masthead--default`
- Navigate to the "Knobs" tab and set the user authenticated status to `authenticated`.
- Ensure that the icon is set to `20px`.

### Expected Display

![Screen Shot 2021-09-21 at 10 46 04 AM](https://user-images.githubusercontent.com/1815714/134192954-cf8da949-76b0-4a38-bfaf-d3376a0a69f3.png)
